src/impl/FormulaCell.test.ts

```diff
@@ -162,6 +162,18 @@ describe('FormulaCell', () => {
         const y = map((x) => ({ equals, y: inc(x) }), x)
         reset(2, x)
         expect(equals).toBeCalled()
-        expect(y.deref().y).toBe(3)
+        expect(deref(y).y).toBe(3)
+    })
+
+    it('should NOT create a formula cell if no input values are cells', () => {
+        for (let i = 1; i <= 10; i++) {
+            const seq = naturalSeq(0, i)
+
+            // @ts-expect-error typescript can not properly derive type in this context
+            const res = formula.apply(null, [sum, ...seq])
```

Comment:
or maybe
```suggestion
            const res = formula.call(null, sum, ...seq)
```  
which one would you prefer? :)

Reply:
Actually, we can do even better:
```typescript
const res = formula(sum, ...seq)
```
Fixed in the latest commit